ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -100,7 +100,7 @@ let prototyped_of_message = function
   | "host", "set_https_only" ->
       Some "22.27.0"
   | "VM", "restart_device_models" ->
-      Some "23.29.0-next"
```

Comment:
This change is unrelated to the rest, please put it into a different commit

Reply:
done